### DISCUSSION TITLE
Question on arctic code vault badge. Can't you never get more?

### DISCUSSION THREAD

**Discussion by solyasa** [community/community#30334] [Other Feature Feedback, Questions, & Ideas] [answered]
I saw that badge was only available to get in 2020.

Can't you never get more?

**Comment by radar-blip** [comment, marked as answer]
No, you cant get that badge anymore. 
Here is some info about all achievement badges: [Achievement Badges](https://github.com/drknzz/GitHub-Achievements)

**Comment by NikhilMath** [comment]
Yeah, its like a rare limited addition badge. Not all Badges are supposed to be accessible for everyone at any moment.
